### ISSUE TITLE
False positive: as-> inside doto

### ISSUE THREAD

**Issue by yuhan0** [borkdude/clj-kondo#506]
**version**
clj-kondo v2019.09.23-alpha-SNAPSHOT

**platform**
MacOS 10.14

**problem**
There seems to be a parsing bug with the `as->` macro inside a `doto` form, see below

**repro**

```clojure
(doto (java.util.ArrayList. [1 2 3])
  (as-> a
    (.addAll a a)))
```

**expected behavior**

No warnings generated

**actual behavior**
```sh
$ clj-kondo --lint scratch.clj
scratch.clj:3:5: error: unsupported binding form (.addAllaa)
linting took 8ms, errors: 1, warnings: 0
```

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/ca765b5ab6ac15e780a8da99946db204aa3a013a

This is a new Mac binary (which also contains the fix for #507: 

https://6389-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.10.05-alpha-SNAPSHOT-macos-amd64.zip
